gitpod-io/gitpod
# Update go.sum files to avoid having them changed when workspace starts

## Description

| Before | After |
| --- | --- |
| <img width="343" alt="image" src="https://user-images.githubusercontent.com/418083/209375895-2c3137fa-0b12-4b76-b991-f8bb4334f560.png"> | <img width="339" alt="image" src="https://user-images.githubusercontent.com/418083/209375595-43cb25b9-101b-4a2a-b0dd-29d01849efe5.png"> |

After these files were committed and pushed, I've reopened the workspace a few times to ensure nothing else was being changed during the startup.

I've also run a [prebuild from this branch](https://gitpod.io/t/gitpod/gitpod/421035b8-f922-4333-9ccd-6cf8f995dfe5), and confirmed the prebuild is not re-changing them.

The hard part will be to get the approval of every code-owner before someone else changes one of those files in the main branch and generates a file conflict. But I'll keep this branch up to date until we merge it.

Internal discussions: [[1](https://gitpod.slack.com/archives/C02EN94AEPL/p1671092590802639)][[2](https://gitpod.slack.com/archives/C02EN94AEPL/p1670586284936429?thread_ts=1670573389.952759&cid=C02EN94AEPL)][[3](https://gitpod.slack.com/archives/C02EN94AEPL/p1670164618788119)][[4](https://gitpod.slack.com/archives/C032A46PWR0/p1671179618175619)]

Previous fix attempt: [[1](https://gitpod.slack.com/archives/C032A46PWR0/p1671179964603239?thread_ts=1671179618.175619&cid=C032A46PWR0)][[2](https://github.com/gitpod-io/gitpod/pull/15311)]

## How to test

- Check if Werft Build succeeds.
- Open a workspace from this PR in gitpod.io (https://gitpod.io/#https://github.com/gitpod-io/gitpod/pull/15497) and confirm there's no file changed when the workspace starts.

## Release Notes

```release-note
NONE
```

## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
